src/neural/encoder.h

```diff
@@ -32,8 +32,11 @@
 
 namespace lczero {
 
+enum FillEmptyHistory {NO, FEN_ONLY, ALWAYS};
+
 // Encodes the last position in history for the neural network request.
 InputPlanes EncodePositionForNN(const PositionHistory& history,
-                                int history_planes);
+                                int history_planes,
+                                FillEmptyHistory fill_empty_history = NO);
```

Comment:
In general it's not recommended to have default parameters. Styleguide disallows them, but we still use them sometimes. But here it doesn't seem to be justified case to use it.